server/utils/userCaseLoads.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Module containing utility functions to do with user caseloads
+ */
+import { User } from '../data/manageUsersApiClient'
```

Comment:
I'm not entirely sure these utility functions to do with user caseloads and user roles warrant being here as re-usable functions, because they are only used by `checkPrisonerInCaseloadMiddleware`, and I'm not sure I can see a future use case where we might need them elsewhere? 🤷‍♂️ 

But it's similar to how HMPPS Prisoner Profile has done it, so I've followed suit.
("Similar" in that they have theirs in a massive `server/utils/utils.ts` file - at least I've broken ours our into smaller modules based on a theme)